src/EncoderDecoder/Huffman/HuffmanEncoderDecoder.cpp

```diff
@@ -0,0 +1,252 @@
+#include "EncoderDecoder/Huffman/HuffmanEncoderDecoder.h"
+
+HuffmanEncoderDecoder::NodeVector HuffmanEncoderDecoder::getFrequencies(
+	const std::string &plaintext) {
+
+	// Initialize frequencies
+	NodeVector frequencies;
+	frequencies.reserve(256);
```

Comment:
Magical 256

Reply:
Also, I think this can be done in one call using vector’s ctor
```C++
vector vec(100);
```

Reply:
> Also, I think this can be done in one call using vector’s ctor
> 
> ```c++
> vector vec(100);
> ```

In this case, I didn't want ctors to be called, but only for space to be reserved.

Reply:
> Magical 256

TODO